NeuroID/NeuroIDClass/Extensions/NIDAdvancedDevice.swift

```diff
@@ -11,27 +11,38 @@ import NeuroIDAdvancedDevice
 public extension NeuroID {
     internal static var deviceSignalService: DeviceSignalService = NeuroIDADV()
 
-    static func start(_ advancedDeviceSignals: Bool) -> Bool {
-        let started = NeuroID.start()
-        
-        if !started {
-            return started
+    static func start(
+        _ advancedDeviceSignals: Bool,
+        completion: @escaping (Bool) -> Void = { _ in }
+    ) -> Bool {
+        let started = NeuroID.start { started in
+            if !started {
+                completion(started)
+                return
+            }
+                   
+            checkThenCaptureAdvancedDevice(advancedDeviceSignals)
         }
-               
-        checkThenCaptureAdvancedDevice(advancedDeviceSignals)
         
+        // this is now inaccurate but keeping for backwards compatibility
```

Comment:
Are there any TODOs that need to be called out with this comment?

Since we are changing the interface, and doing a rev of the SDK, is backwards compatibility needed here?

Reply:
This is more of an internal note for us to decide before merging. Ed and I were going back and forth on if its needed because its a minor rev

Reply:
It was decided to remove the returned value and only use the callback. This is a breaking change for customers however it is a minor bump so breakage is allowed. Additionally this will prevent compiling so a customer cannot ship without changing if they update to the latest version.